File: go/prob1.go
package main

import "fmt"

func main() {
	n := 10

	sum := 0
	for i := 1; i < n; i++ {
		if (i%3 == 0) || (i%5 == 0) {
			sum += i
		}
	}
	fmt.Printf("Answer for problem 1 is %d\n", sum)
}

File: prob1.py

'''
https://projecteuler.net/problem=1
If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9. The sum of these multiples is 23.

Find the sum of all the multiples of 3 or 5 below 1000.
'''

sum = 0

print("Please enter an maximum number that you want")

N = int(input())

for i in range(1, N):
    if (i % 3 == 0) or (i % 5) == 0:
        sum += i
print(f"Sum of all multiples of 3 or 5 below {N} is {sum}")
